src/main/java/com/dataloom/datastore/edm/controllers/EdmController.java

```diff
@@ -775,10 +775,19 @@ public AssociationType getAssociationTypeById( @PathVariable( ID ) UUID associat
 
     @Override
     @RequestMapping(
-            path = ASSOCIATION_TYPE_PATH + ID_PATH + DETAILED_PATH,
-            method = RequestMethod.GET,
-            produces = MediaType.APPLICATION_JSON_VALUE )
+        path = ASSOCIATION_TYPE_PATH + ID_PATH + DETAILED_PATH,
+        method = RequestMethod.GET,
+        produces = MediaType.APPLICATION_JSON_VALUE )
     public AssociationDetails getAssociationDetailsForType( @PathVariable( ID ) UUID associationTypeId ) {
         return modelService.getAssociationDetails( associationTypeId );
     }
+
+    @Override
+    @RequestMapping(
+        path = ASSOCIATION_TYPE_PATH + ID_PATH + AVAILABLE_PATH,
+        method = RequestMethod.GET,
+        produces = MediaType.APPLICATION_JSON_VALUE )
+    public Iterable<EntityType> getAvailableAssociationTypesForEntityType( @PathVariable( ID ) UUID entityTypeId ) {
```

Comment:
The name of this method is confusing. The method is returning `Iterable<EntityType>`, which is a collection of `EntityType`, but the name suggest that it should be returning a collection `AssociationType`.